src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const LONG_TERM = 7;
+  const MID_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const MID_TERM_DISCOUNT = 20;
+  let amount = 0;
+
+  if (days < MID_TERM) {
+    amount = rent * days;
+  }
```

Comment:
The condition `days < MID_TERM` correctly handles short-term rentals without any discount. This part of the logic is implemented correctly.